src/overlays/actors/ovl_En_Bal/z_en_bal.c

```diff
@@ -15,7 +15,79 @@ void EnBal_Destroy(Actor* thisx, PlayState* play);
 void EnBal_Update(Actor* thisx, PlayState* play);
 void EnBal_Draw(Actor* thisx, PlayState* play);
 
-#if 0
+typedef enum {
+    /* 0 */ TINGLE_EYETEX_OPEN,
+    /* 1 */ TINGLE_EYETEX_CLOSED
+} EnBalEyeTexture;
+
+typedef enum {
+    /*  0 */ TINGLE_ANIM_FLOAT_DRAW,
+    /*  1 */ TINGLE_ANIM_FALL_LOOP,
+    /*  2 */ TINGLE_ANIM_FALL_ONCE,
+    /*  3 */ TINGLE_ANIM_LAND,
+    /*  4 */ TINGLE_ANIM_TALK_TWIST,
+    /*  5 */ TINGLE_ANIM_TALK_IDLE,
+    /*  6 */ TINGLE_ANIM_MAGIC_FULL,
+    /*  7 */ TINGLE_ANIM_HAPPY_DANCE_LOOP,
+    /*  8 */ TINGLE_ANIM_HAPPY_DANCE_ONCE,
+    /*  9 */ TINGLE_ANIM_MAGIC_SHORT,
+    /* 10 */ TINGLE_ANIM_WAIT,
+    /* 11 */ TINGLE_ANIM_SPIN,
+    /* 12 */ TINGLE_ANIM_HIDE_FACE,
+    /* 13 */ TINGLE_ANIM_CONFETTI,
+    /* 14 */ TINGLE_ANIM_MAX
+} EnBalAnimation;
```

Comment:
```suggestion
} TingleAnimation;
```

I've also been putting these animation enum definitions right above `sAnimationInfo` in my cleanup, but that's up to you